src/components/Widgets/TaskPropertiesWidget/Messages.js

```diff
@@ -0,0 +1,16 @@
+import { defineMessages } from 'react-intl'
+
+/**
+ * Internationalized messages for use with TaskMoreOptionsWidget
+ */
+export default defineMessages({
+  label: {
+    id: "Widgets.TaskPropertiesWidget.label",
+    defaultMessage: "Task Properites",
```

Comment:
The word "Properties" has a typo in the label message. Also, can you update the comment to refer to TaskPropertiesWidget instead of TaskMoreOptionsWidget?